certificate-manager/src/test/java/io/strimzi/certs/OpenSslCertManagerTest.java

```diff
@@ -208,30 +235,25 @@ private void testGenerateSignedCert(File caKey, File caCert, Subject caSbj, File
 
         ssl.addKeyAndCertToKeyStore(caKey, caCert, "ca", keyStore, keyStorePassword);
 
-        CertificateFactory cf = CertificateFactory.getInstance("X.509");
-        Certificate c = cf.generateCertificate(new FileInputStream(cert));
-        Certificate ca = cf.generateCertificate(new FileInputStream(caCert));
+        X509Certificate c = loadCertificate(cert);
+        assertCaCertificate(c, false);
```

Comment:
I am missing something, isn't this going to fail anyway because of the `false` parameter you are passing in?

Reply:
The false means we're asserting that the issued cert is not a CA cert (i.e. _does_ not have `CA: true`). This is in a test about the issued certs, not the CA certs. The issued certs should not have `CA: true` because they're strictly client certs.